data/translations/es/state-data.json

```diff
@@ -324,27 +325,27 @@
     "registration_link": "https://www.nvsos.gov/SOSVoterServices/start.aspx",
     "registration_link_language_selection": "true",
     "more_info_link": "https://www.nvsos.gov/sos/elections/voters/registering-to-vote",
-    "ip_deadline": "hasta el cuarto martes antes del día de las elecciones",
-    "online_deadline": "hasta el jueves antes de que empiece la votación anticipada",
+    "ip_deadline": "martes, 08 de noviembre de 2022",
+    "online_deadline": "martes, 03 noviembre de 2022",
     "mail_received_deadline": "",
-    "mail_postmarked_deadline": "debe tener el sello del correo a más tardar el cuarto martes antes del día de las elecciones",
+    "mail_postmarked_deadline": "debe tener el sello del correo martes, 11 de octubre de 2022",
     "confirm_registration_link": "https://www.nvsos.gov/votersearch/"
   },
   "nh": {
     "hp_link": "https://sos.nh.gov/home",
     "more_info_link": "https://sos.nh.gov/elections/voters/register-to-vote/",
-    "ip_deadline": "hasta el mismo día de las elecciones",
+    "ip_deadline": "martes, 08 de noviembre de 2022",
     "confirm_registration_link": "https://app.sos.nh.gov/Public/PartyInfo.aspx"
   },
   "nj": {
     "hp_link": "https://www.state.nj.us/state/elections/index.shtml",
     "registration_link": "https://voter.svrs.nj.gov/register",
     "registration_link_language_selection": "true",
     "more_info_link": "https://nj.gov/state/elections/voter-registration.shtml",
-    "ip_deadline": "hasta 21 días antes del día de las elecciones",
-    "online_deadline": "hasta 21 días antes del día de las elecciones",
-    "mail_received_deadline": "a más tardar debe ser recibida 21 días antes del día de las elecciones",
-    "mail_postmarked_deadline": "",
+    "ip_deadline": "martes, 18 de octubre de 2022",
+    "online_deadline": "martes, 18 de octubre de 2022",
+    "mail_received_deadline": "",
```

Comment:
Updated from received to postmarked. Please check this. Also remove empty deadline properties.

Reply:
confirmed